OpenEdXMobile/src/main/java/org/edx/mobile/view/WebViewDiscoverFragment.java

```diff
@@ -97,7 +97,9 @@ public void onUserNotLoggedIn(@NonNull String courseId, boolean emailOptIn) {
 
     @Override
     public void onSaveInstanceState(Bundle outState) {
-        outState.putString(INSTANCE_CURRENT_DISCOVER_URL, binding.webview.getUrl());
+        if (URLUtil.isValidUrl(binding.webview.getUrl())) {
```

Comment:
It deserves Java docs, why we are saving this url

Reply:
done.